### PR TITLE
v0.2.0 of sphinxcontrib-video initial conda-forge package added

### PR DESCRIPTION
**Pull request by ReimarBauer**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ReimarBauer** [comment]
@conda-forge/help-python ready for review!

further maintainers are welcome  :)

File: conda_build_config.yaml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-MACOSX_DEPLOYMENT_TARGET:  # [osx]
-  - 11.0                   # [osx]
\ No newline at end of file
+c_stdlib_version:  # [osx]
+  - 11.0           # [osx]
```
Comment (ocefpaf) [inline conda_build_config.yaml:2]: Are you sure you need this file? The package is noarch, right?

Comment (ReimarBauer) [reply conda_build_config.yaml:2]: it is noarch, the linter complained, that's why I changed it.

```
For recipes/sphinxcontrib-video:

In your conda_build_config.yaml, please change the name of MACOSX_DEPLOYMENT_TARGET, to c_stdlib_version!
```

Comment (xhochy) [reply conda_build_config.yaml:2]: Do you need the deployment target? Normally noarch packages don't need any of this.

Comment (ReimarBauer) [reply conda_build_config.yaml:2]: When the solution is to remove it, I do that.

The hint could have been a bit more detailed.

Comment (ocefpaf) [reply conda_build_config.yaml:2]: Well, the hint assumed you added that file b/c you neeed it. Our lints are not smart enough to know everything about your package. The question is, why you added that in the first place? If you don't know you need extra config, try without them first.

Comment (xylar) [reply conda_build_config.yaml:2]: This is also happening to me in #26412

Comment (ocefpaf) [reply conda_build_config.yaml:2]: Ah. Good to know!

Comment (xylar) [inline conda_build_config.yaml:2]: Are you sure you want to be changing things outside of `recipes` here?  Wouldn't it be better to do this as its own PR?

Comment (xylar) [reply conda_build_config.yaml:2]: @ocefpaf?

Comment (ocefpaf) [reply conda_build_config.yaml:2]: Yep. Let's just revert this and ignore the linter for now. Whoever made this change in conda-smithy probably just forgot to update it here. I'm out of the loop in this change so I'd rather not touch it for now.

Comment (xylar) [reply conda_build_config.yaml:2]: @xhochy, it looks like you were involved in the PR that added this file:
https://github.com/conda-forge/staged-recipes/pull/24549
Any insight?

In any case, nothing that needs to be discussed here...

**Comment by ReimarBauer** [comment]
@ocefpaf @xylar I have undone that change.

**Comment by ReimarBauer** [comment]
I copied the version from branch main over the old. This seems not to have a newline ending, wanted?

**Comment by ReimarBauer** [comment]
Thx, 

@ocefpaf I've just seen that on the new modules page https://anaconda.org/conda-forge/sphinxcontrib-video

The  description I provided is missing.  Was there a syntax change?

**Comment by ocefpaf** [comment]
> The description I provided is missing. Was there a syntax change?

I don't know how that is populated. To be honest, I don't even look at the anaconda platform for package info, only the feedstock. I'd ask in the element conda-forge channel and hope that someone from Anaconda answers.
